package.json

```diff
@@ -74,32 +74,26 @@
     "bufferutil": "4.0.1",
     "chance": "1.1.4",
     "compression": "1.7.4",
-    "dclassify": "1.1.2",
-    "event-stream": "4.0.1",
     "express": "4.17.1",
     "ftp-srv": "4.3.4",
     "get-folder-size": "2.0.1",
     "getos": "3.2.0",
     "glob-to-regexp": "0.4.1",
-    "googleapis": "45.0.0",
+    "googleapis": "59.0.0",
     "heapdump": "0.3.15",
     "http-proxy-agent": "4.0.1",
     "https-proxy-agent": "5.0.0",
-    "ini": "1.3.5",
     "ip": "1.1.5",
     "jsonwebtoken": "8.5.1",
     "linux-blockutils": "0.1.0",
     "lodash": "4.17.19",
     "mime": "2.4.4",
     "minimist": "1.2.5",
-    "mkdirp": "0.5.1",
```

Comment:
still be required and used in create_path in fsutils

Reply:
yes, I reverted the change I made and missed adding it back to the package.json
We will be able to remove it from fs_utils by replacing it with fs.mkdir(dir, { recursive: true }, callback)
We can then remove the mode as the default mode is 0o777